### PR TITLE
Update README for 1.0.0 version

### LINKED ISSUE #107: Update the README
- Some badges don't work
- The sample code is outdated.
- Should add a link to the sample(s)
- Should add a blurb about our own use in the Android / Android TV apps as example implementations

### LINKED ISSUE #195: Rename the repository to jellyfin-sdk-kotlin
We've had a discussion about renaming this project before. A few names came up like "jellyfin-sdk-kotlin" and "jellyfin-apiclient-kotlin".

Since the project is entirely written in Kotlin now I think it's better to not use the term "java" in the name. I'd like to start using Kotlin Multiplatform at some point which allows us to target not only the JVM but also JavaScript, WebAssembly and iOS (just to name a few).

I'd rather rename the repository before releasing 1.0.0, although this only matters for the metadata because our package and group use "org.jellyfin.apiclient".

The new readme in #193 already uses the new name 😉 

edit (2021-03-19): the initial plan (written above) was to rename to "apiclient" but we decided "sdk" makes more sense. See my comment below for the new plan.

### PR DESCRIPTION
**Pull request by nielsvanvelzen**
Closes #107  
Closes #195

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,152 +36,165 @@
 
 ---
 
-This library allows Java and Android applications to easily access Jellyfin servers.
-The dependencies are modular and can easily be swapped out with alternate implementations when desired.
+The Jellyfin Kotlin API Client is a library implementing the Jellyfin API to easily access servers.
+It is currently available for the JVM and Android targets.
```
Comment (thornbill) [inline README.md:40]:
```suggestion
It is currently available for the JVM and Android platforms.
```

**Review by thornbill — COMMENTED** [review]
Looks great! Only one minor suggestion.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,20 @@
...
+<a href="https://github.com/jellyfin/jellyfin-sdk-kotlin/releases">
+<img alt="Current Release" src="https://img.shields.io/github/release/jellyfin/jellyfin-sdk-kotlin.svg"/>
 </a>
-<a href="https://github.com/jellyfin/jellyfin-apiclient-java/releases">
-<img alt="Current Release" src="https://img.shields.io/github/release/jellyfin/jellyfin-apiclient-java.svg"/>
-</a>
-<a href="https://bintray.com/jellyfin/jellyfin-apiclient-java/jellyfin-apiclient-java">
-<img alt="Bintray Release" src="https://img.shields.io/bintray/v/jellyfin/jellyfin-apiclient-java/jellyfin-apiclient-java.svg"/>
+<a href="https://bintray.com/jellyfin/jellyfin-sdk-kotlin/jellyfin-sdk-kotlin">
+<img alt="Bintray Release" src="https://img.shields.io/bintray/v/jellyfin/jellyfin-sdk-kotlin/jellyfin-sdk-kotlin.svg"/>
```
[the remark this answers is not in the document]
Comment (nielsvanvelzen) [reply README.md:17]: Replacing it with a maven badge

File: README.md [hunk 3]
```diff
@@ -29,162 +26,175 @@
...
-		// Authentication succeeded
-	}
+// Use access token in api instance
+api.accessToken = authenticationResult.accessToken
 
-	override fun onError(error: Exception) {
-		// Authentication failed
-	}
-})
+println(authenticationResult.accessToken)
```
[the remark this answers is not in the document]
Comment (nielsvanvelzen) [reply README.md:144]: Changed it to not print access token

**Comment by nielsvanvelzen** [comment]
Thanks for the thorough review @Maxr1998, I noticed some lines still using the "api client" term so I updated those as well.

File: README.md [hunk 4]
```diff
@@ -1,23 +1,20 @@
...
+<a href="https://github.com/jellyfin/jellyfin-sdk-kotlin/releases">
+<img alt="Current Release" src="https://img.shields.io/github/release/jellyfin/jellyfin-sdk-kotlin.svg"/>
 </a>
-<a href="https://github.com/jellyfin/jellyfin-apiclient-java/releases">
-<img alt="Current Release" src="https://img.shields.io/github/release/jellyfin/jellyfin-apiclient-java.svg"/>
-</a>
-<a href="https://bintray.com/jellyfin/jellyfin-apiclient-java/jellyfin-apiclient-java">
-<img alt="Bintray Release" src="https://img.shields.io/bintray/v/jellyfin/jellyfin-apiclient-java/jellyfin-apiclient-java.svg"/>
+<a href="https://search.maven.org/search?q=org.jellyfin.sdk">
+<img alt="Bintray Release" src="https://img.shields.io/maven-central/v/org.jellyfin.sdk/jellyfin-core.svg"/>
```
Comment (Maxr1998) [inline README.md:17]:
```suggestion
<img alt="Maven Release" src="https://img.shields.io/maven-central/v/org.jellyfin.sdk/jellyfin-core.svg"/>
```

:eyes:

**Review by Maxr1998 — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (nielsvanvelzen) [reply README.md:17]: I'm stupid

**Review by Maxr1998 — APPROVED** [review]

**Pull request merged**